tests/test_hf_v_mpt.py

```diff
@@ -165,7 +165,7 @@ def test_compare_hf_v_mpt(attn_impl: str, dropout: float, alibi: bool,
     print(f'{hf_model_fwd.mean().item() = }\n{model_fwd.mean().item() = }')
     if hf_model_fwd.mean().allclose(model_fwd.mean()):
         warn_msg = f'WARNING: model_fwd ({model_fwd}) and hf_model_fwd ({hf_model_fwd}) are very close at init.'
-        raise warnings.warn(warn_msg)  # type: ignore
+        raise RuntimeError(warn_msg)
```

Comment:
The previous line `raise warnings.warn(..)` is ambigious as to the author's intent. Do we want to raise an exception here or emit a warning? Since this is a test, I assume exception.

Reply:
@vchiley Which was this supposed to be?

Reply:
Was I dumb enough to raise a warning. lololol

This is supposed to be a warning since it could cause a "false pass" of the test.